docs/api/modal.md

```diff
@@ -102,11 +96,13 @@ The `presentingElement` property accepts a reference to the element that should
 
 The `swipeToClose` property can be used to control whether or not the card modal can be swiped to close.
 
-TODO: Playground Example
+:::note
+The card display style is only available on iOS.
+:::
 
-#### Custom sizes
+import CardExample from '@site/static/usage/modal/card/basic/index.md';
 
-TODO: Playground Example
+<CardExample />
```

Comment:
On the JS Stackblitz, if I click the Close button (rather than swiping to close) I get an error (`modal is not defined`).